src/pages/tasks/NewTaskPage.js

```diff
@@ -72,35 +73,56 @@ function NewTaskPage(props) {
 
     const isAllowedToCreateTask = useMemo(() => _.isEmpty(parentReport) || ReportUtils.isAllowedToComment(parentReport), [parentReport]);
 
+    useEffect(() => {
+        setErrorMessage('');
+
+        // We only set the parentReportID if we are creating a task from a report
+        // this allows us to go ahead and set that report as the share destination
+        // and disable the share destination selector
+        if (props.task.parentReportID) {
+            Task.setShareDestinationValue(props.task.parentReportID);
+        }
+    }, [props.task.parentReportID]);
+
     useEffect(() => {
         setErrorMessage('');
 
         // If we have an assignee, we want to set the assignee data
         // If there's an issue with the assignee chosen, we want to notify the user
         if (props.task.assignee) {
+            const user = _.find(props.personalDetails, ({login}) => login === props.task.assignee);
+
+            if (lodashGet(user, 'accountID') && user.accountID !== props.task.assigneeAccountID) {
```

Comment:
I guess a default value should be
```suggestion
            if (lodashGet(user, 'accountID', null) && user.accountID !== props.task.assigneeAccountID) {
```

Reply:
lodash's get defaults to `undefined` if it's not provided

Reply:
Yes, it does. However, it is better to assign some empty value like `0`, `''`, `null`, etc. Mostly it has a default value across this project.